gems/quilt_rails/lib/generators/quilt/install_generator.rb

```diff
@@ -8,7 +8,24 @@ class InstallGenerator < Rails::Generators::Base
 
     def install_js_dependencies
       say "Installing @shopify/react-server and @shopify/sewing-kit dependencies"
-      system("yarn add @shopify/sewing-kit @shopify/react-server") unless Rails.env.test?
+      system("yarn add "\
```

Comment:
I think we might want to lock these to a version. What do you think?

Reply:
definitely for react and typescript.

Reply:
Agreed, but stuff is busted, so it's a tomorrow problem: https://github.com/Shopify/quilt/issues/1040